my-app/src/dataItem.css

```diff
@@ -0,0 +1,104 @@
+.dataItemWrapper {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-around;
+  align-items: center;
+  margin: auto;
+
+  border: solid 2px darkgray;
+  background-color: whitesmoke;
+  color:#313237;
```

Comment:
Move all repeatable colors to the CSS variables.

[More info.](https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties)